new-backoffice/getting-started/installing-preview-builds/README.md

```diff
@@ -74,7 +74,13 @@ With that, we've successfully installed the latest build of Umbraco! All we have
 dotnet new umbraco -n MyAwesomeNightlySite
 ```
 
-This creates a folder named "MyAwesomeNightlySite" on your system. You can open it in your IDE (like Rider) or use the command line to start working with it. If you choose to use the command line first you have to change the directory to `MyAwesomeNightlySite`.
+This creates a folder named "MyAwesomeNightlySite" on your system. You can open it in your IDE (like Rider) or use the command line to start working with it. 
+
+If you choose to use the command line first you have to change the directory to `MyAwesomeNightlySite`.
```

Comment:
In this case it seems you are sharing an example how to achieve this via GitBash, therefore it should be mentioned. Otherwise in the example it is specified command line and then the example should be just `cd MyAwesomeNightlySite` instead of `cd .\MyAwesomeNightlySite\`
```suggestion
If you choose to use the command line first you have to change the directory to `MyAwesomeNightlySite`. Below you can find an example of how to achieve this using GitBash.
```

Reply:
Hey @alina-tincas, It can be changed to just be cd MyAwesomeNightlySite with no backslashes to keep the same examples going. I think saying using gitbash would be more confusing for new people who are just using the command line. I can update the PR. :)